### PR TITLE
Reduce output of cell_categorization_02

### PR DESCRIPTION
**Pull request by masterleinad**
This PR reduces the output size of `matrix_free\cell_categorization_02`.
~to about the same as a bit `matrix_free\cell_categorization`.~ 
~Additionally, it replaces the (default) output by the one generated with SSE2 in accordance with `matrix_free\cell_categorization_02`.~
~This should fix most of the failures of this test reported in [`CDash`](https://cdash.kyomu.43-1.org/testSummary.php?project=1&name=matrix_free%2Fcell_categorization_02.mpirun%3D1.debug&date=2018-08-04).~
~Of course, we also need `AVX` and `AVX512` versions for this test.~

### CODE REVIEW

**Comment by masterleinad** [comment]
@davydden Do you want to push your version here as well? I guess you have AVX available, don't you?

**Comment by davydden** [comment]
What we can do is remove printing of cells. There is already assert-throw that makes sure all filled cells in the block have the same material id. So the test can just output “Ok”

**Comment by masterleinad** [comment]
Yes, I thought about this as well, but didn't want to change the test (or its output) that dramatically without your consent. I am happy to change that know. It is clearly advantageous when we don't need 4 output files.

**Comment by kronbichler** [comment]
I agree on that route. I think it makes sense to have one or two test cases where the order is checked (the old `cell_categorization` test, but the less architecture-specific output variants we need, the better.

**Comment by masterleinad** [comment]
I removed all the output apart from one final `OK` now.

File: tests/matrix_free/cell_categorization_02.cc [hunk 1]
```diff
@@ -79,7 +79,7 @@ test()
           cell->set_material_id(1);
       }
 
-    tria.refine_global(4);
```
Comment (davydden) [inline tests/matrix_free/cell_categorization_02.cc]: you can keep this ow, not that it will matter, but to be on the safe side.

**Comment by masterleinad** [comment]
I am using four refinements again. It should really not matter at all.

**Pull request merged**